services/src/main/java/io/grpc/services/HealthServiceImpl.java

```diff
@@ -29,28 +29,35 @@
 import io.grpc.stub.StreamObserver;
 import java.util.HashMap;
 import java.util.IdentityHashMap;
+import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.GuardedBy;
 
 final class HealthServiceImpl extends HealthGrpc.HealthImplBase {
 
+  private static final Logger logger = Logger.getLogger(HealthServiceImpl.class.getName());
+
   // Due to the latency of rpc calls, synchronization of the map does not help with consistency.
   // However, need use ConcurrentHashMap to allow concurrent reading by check().
-  private final ConcurrentHashMap<String, ServingStatus> statusMap =
-      new ConcurrentHashMap<String, ServingStatus>();
+  private final Map<String, ServingStatus> statusMap = new ConcurrentHashMap<>();
```

Comment:
This change is needed to make Animal Sniffer not get mad.    ConcurrentHashMap.keySet is apparently a Java 8 method.